### PR TITLE
Update admin-submission.md

### PR DESCRIPTION
**Pull request by msbemba**
updated Mark as and notify

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6511

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 1]
```diff
@@ -72,6 +72,20 @@ For other ways to submit email messages, URLs, and attachments to Microsoft, see
    > [!NOTE]
    > File and URL submissions are not available in the clouds that do not allow for data to leave the environment. The ability to select File or URL will be greyed out.
 
+## Notify users from within the portal
+
+1. In the Microsoft 365 Defender portal, go directly to the **Submissions** page: [https://security.microsoft.com/reportsubmission](https://security.microsoft.com/reportsubmission).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
1. In the Microsoft 365 Defender portal, go directly to the [**Submissions**](https://security.microsoft.com/reportsubmission) page.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 2]
```diff
@@ -72,6 +72,20 @@ For other ways to submit email messages, URLs, and attachments to Microsoft, see
...
+
+2. Click **User reported messages**, and then select the message you want to mark and notify.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
2. Select **User reported messages**, and then select the message you want to mark and notify.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 3]
```diff
@@ -72,6 +72,20 @@ For other ways to submit email messages, URLs, and attachments to Microsoft, see
...
+
+3. Select the **Mark as and notify** drop-down, and then select **No threats found**, **Phishing**, or **Junk**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
3. Select the **Mark as and notify** drop-down, and then select **No threats found** > **Phishing** or **Junk**.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 4]
```diff
@@ -72,6 +72,20 @@ For other ways to submit email messages, URLs, and attachments to Microsoft, see
...
+
+   > [!div class="mx-imgBorder"]
+   > ![Send messages from portal.](../../media/admin-review-send-message-from-portal.png)
+
+The reported message will be marked as either false positive or false negative, and an email will be automatically sent from within the portal notifying the user who reported the message.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md:86]:
```suggestion
The reported message will be marked as either false positive or false negative, and an email will be sent automatically from within the portal, notifying the user who reported the message.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 5]
```diff
@@ -72,6 +72,20 @@ For other ways to submit email messages, URLs, and attachments to Microsoft, see
...
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
The reported message will be marked as either false positive or false negative and an email will be sent automatically from within the portal to notify the user who reported the message.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**